firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/metadata/MetaDataStore.java

```diff
@@ -231,8 +228,9 @@ private static String keysDataToJson(final Map<String, String> keyData) {
   }
 
   private static List<RolloutAssignment> jsonToRolloutsState(String json) throws JSONException {
-    final List<RolloutAssignment> rolloutsState = new ArrayList<RolloutAssignment>();
-    final JSONArray dataArray = new JSONArray(json);
+    JSONObject object = new JSONObject(json);
+    JSONArray dataArray = object.getJSONArray("rolloutsState");
```

Comment:
Thinking: the name of this field (`rolloutsState`) should be imported from the Interop SDK, or the naming and encoder logic should live in the Crashlytics SDK

Reply:
Added name to RolloutAssignmentList.